var/spack/repos/builtin/packages/octopus/package.py

```diff
@@ -31,26 +45,62 @@ class Octopus(Package, CudaPackage):
             description='Compile with Netcdf')
     variant('arpack', default=False,
             description='Compile with ARPACK')
+    variant('cgal', default=False,
+            description='Compile with CGAL library support')
+    variant('pfft', default=False,
+            description='Compile with PFFT')
+    # poke here refers to https://gitlab.e-cam2020.eu/esl/poke
+    # variant('poke', default=False,
+    #         description='Compile with poke (not available in spack yet)')
+    variant('python', default=False,
+            description='Activates Python support')
+    variant('likwid', default=False,
+            description='Compile with likwid')
+    variant('libvdwxc', default=False,
+            description='Compile with libvdwxc')
+    variant('libyaml', default=False,
+            description='Compile with libyaml')
+    variant('elpa', default=False,
+            description='Compile with ELPA')
+    variant('nlopt', default=False,
+            description='Compile with nlopt')
+    variant('debug', default=False,
+            description='Compile with debug flags')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+    depends_on('m4',       type='build')
 
     depends_on('blas')
     depends_on('gsl@1.9:')
     depends_on('lapack')
-    depends_on('libxc@2.0:2', when='@:5')
-    depends_on('libxc@2.0:3', when='@6:7')
-    depends_on('libxc@2.0:4', when='@8:9')
-    depends_on('libxc@3:5.0.0', when='@10:')
+    depends_on('libxc@2:2.99', when='@:5.99')
```

Comment:
I think you don't need the `.99` in the version ranges now.

Reply:
So I could replace line 78 with `depends_on('libxc@2:2.99', when='@:5')` ?

Reply:
See: https://github.com/spack/spack/commit/b9e72557e8b52b63244500f226afda0efe798df2#diff-b8373d30b3a141c495c2281273ee6184fc513413142afaf2adac1f406cd6b0d7

Reply:
Thank you. I have implemented this (https://github.com/spack/spack/pull/30321/commits/492d0ce8668d98de9db59352a3347c51b2a49ed8).